src/components/TimePicker.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+    <v-row justify="center" align="center" class="ma-0 pa-0">
+        <div class="centered" />
+        <v-col cols="5" class="swiperHolder pa-0 d-flex">
+            <swiper ref="hourSwiper" :options="swiperOption">
+                <swiper-slide v-for="(hour, i) in hours" :key="i">
+                    <span class="hour">{{ hour }}</span>
+                </swiper-slide>
+            </swiper>
+        </v-col>
+        <v-col cols="5" class="swiperHolder pa-0 d-flex">
+            <swiper ref="minuteSwiper" :options="swiperOption">
+                <swiper-slide v-for="(minute, i) in minutes" :key="i">
+                    <span class="hour">{{ minute }}</span>
+                </swiper-slide>
+            </swiper>
+        </v-col>
+    </v-row>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from 'vue-property-decorator';
+import SwiperClass, { SwiperOptions } from 'swiper';
+import { Swiper, SwiperSlide } from 'vue-awesome-swiper';
+
+@Component({
+    components: {
+        Swiper,
+        SwiperSlide,
+    },
+})
+export default class AttractionTime extends Vue {
+    @Prop() currentDate?: Date;
+
+    private hours = [
+        '01',
+        '02',
+        '03',
+        '04',
+        '05',
+        '06',
+        '07',
+        '08',
+        '09',
+        '10',
+        '11',
+        '12',
+        '13',
+        '14',
+        '15',
+        '16',
+        '17',
+        '18',
+        '19',
+        '20',
+        '21',
+        '22',
+        '23',
+        '00',
+    ];
+
+    private minutes = [
+        '00',
+        '05',
+        '10',
+        '15',
+        '20',
+        '25',
+        '30',
+        '35',
+        '40',
+        '45',
+        '50',
+        '55',
+    ];
+
+    data(): {
+        activeHour: string;
+        activeMinute: string;
+        swiperOption: SwiperOptions;
+    } {
+        return {
+            activeHour: '08',
+            activeMinute: '15',
+            swiperOption: {
+                direction: 'vertical',
+                effect: 'coverflow',
+                coverflowEffect: {
+                    rotate: 0,
+                    stretch: 0,
+                    depth: 100,
+                    modifier: 1,
+                    slideShadows: false,
+                },
+                centeredSlides: true,
+                slidesPerView: 6,
+                slideToClickedSlide: true,
+            },
+        };
+    }
+
+    created(): void {
+        this.$data.activeHour = this.$props.currentDate.getHours().toString();
```

Comment:
we talked about removing this data (activeMinutes) - but I'm not sure it the right solution because you use this prop many times